presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -114,6 +115,11 @@
 
     private final List<OperatorStats> operatorSummaries;
 
+    /**
+     * RuntimeStats aggregated at the query level including the metrics exposed in every task and every operator.
+     */
```

Comment:
javadocs are used for generating documents for public interfaces. For private variables, we can just use comments: "// RuntimeStats aggregated ..."

Reply:
Good to know. Updated. Thanks.